db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/TernaryOperatorNode.java

```diff
@@ -84,8 +84,9 @@
 	public static final int TIMESTAMPADD = 4;
 	public static final int TIMESTAMPDIFF = 5;
 	public static final int REPLACE = 6;
-	static final String[] TernaryOperators = {"trim", "LOCATE", "substring", "like", "TIMESTAMPADD", "TIMESTAMPDIFF", "replace"};
-	static final String[] TernaryMethodNames = {"ansiTrim", "locate", "substring", "like", "timestampAdd", "timestampDiff", "replace"};
+	public static final int STRIP = 7;
```

Comment:
This change in this module does not seem necessary. I don't see a case where the operator type will be set to STRIP, as in sqlgrammer.jj, STRIP is eventually generated as a TrimOperatorNode. Please correct me if I missed something.

Reply:
Good catch. The STRIP function returns TrimOperatorNode node type with TernaryOperatorNode.TRIM operator, which is the same as TRIM. STRIP related stuff is unnecessary here.